app/page.tsx

```diff
@@ -23,35 +19,25 @@ export default async function Index() {
   const isSupabaseConnected = canInitSupabaseClient()
 
   return (
-    <div className="flex-1 w-full flex flex-col gap-20 items-center">
-      <nav className="w-full flex justify-center border-b border-b-foreground/10 h-16">
-        <div className="w-full max-w-4xl flex justify-between items-center p-3 text-sm">
-          <DeployButton />
-          {isSupabaseConnected && <AuthButton />}
-        </div>
+    <div
+      className={[
+        'flex flex-col items-center justify-center flex-1',
+        'w-full',
+      ].join(' ')}
+    >
+      <nav
+        className={[
+          'flex flex-col items-center justify-center flex-1',
+          'w-full',
+        ].join(' ')}
+      >
+        <p>Gridiron Survivor</p>
+        {isSupabaseConnected ? (
+          <AuthButton />
+        ) : (
+          'Not connected to Supabase. Ensure the NEXT_SUPABASE_URL and NEXT_SUPABAE_ANON_KEY are set in your .local.env file.'
```

Comment:
You should only show this error message in dev. Showing this in prod is confusing and also reveals information about your tech stack to end users unnecessarily

Reply:
Agree. We should have a generic error here instead. Per PCI compliance, we should not be specific what went wrong. Our internal logs should tell us this, not the public.

Reply:
Fixed.